docs/contracts/abstract-contracts.rst

```diff
@@ -3,18 +3,19 @@
 .. _abstract-contract:
 
 ******************
-Abstract Contracts
+Abstract Contract'lar
 ******************
 
-Contracts must be marked as abstract when at least one of their functions is not implemented or when
-they do not provide arguments for all of their base contract constructors.
-Even if this is not the case, a contract may still be marked abstract, such as when you do not intend
-for the contract to be created directly. Abstract contracts are similar to :ref:`interfaces` but an
-interface is more limited in what it can declare.
+Contractlar en azından bir fonksiyonlarının kodu yazılmadığında veya temel contractlarının
+constructor'larına argüman sağlamadıklarında abstract olarak tanımlanmalıdır.
```

Comment:
Sözleşmeler, işlevlerinden en az biri uygulanmadığında veya bütün temel sözleme yapıcılar için argüman sağlamadığında abstract olarak işaretlenmelidir.